### PR TITLE
Presentation

### PR DESCRIPTION
**Pull request by jassalss**
This is PR for the presentation branch after a massive refactoring as requested. 

GIT and GITHUB start acting funny, version of the branch on my computer was different than the remote. Even pushing from the local branch was not changing the remote. I  had to delete remote and publish it again from the local branch.

Please review the changes and share your feedback.

Thanks

### CODE REVIEW

File: client/src/Component/EventBodyComponent/EventBodyComponent.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import PropTypes from "prop-types";
+import {
+  Grid,
+  Box,
+  Typography,
+  Tab,
+  Tabs,
+  AppBar,
+  makeStyles,
+} from "@material-ui/core";
```
Comment (ThinkTink) [inline client/src/Component/EventBodyComponent/EventBodyComponent.js:11]: :+1:

File: client/src/Component/EventBodyComponent/EventBodyComponent.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  tab_content: {
+    margin: "auto",
+    maxWidth: "80%",
+  },
+  appbar: {
+    backgroundColor: "rgb(247, 247, 247)",
+  },
+  colorForTabs: {
+    color: "primary",
+  },
```
Comment (ThinkTink) [inline client/src/Component/EventBodyComponent/EventBodyComponent.js]: I don't think this is actually used?

File: client/src/Component/EventBodyComponent/EventBodyComponent.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+}
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+const EventBodyComponent = () => {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
```
Comment (ThinkTink) [inline client/src/Component/EventBodyComponent/EventBodyComponent.js]: Let's import useState for this file too

**Comment by ThinkTink** [comment]
Wow, this was a bigger refactor than I realized. Good work on it! Pointed out just a few more small things

File: client/src/Component/MeetingComponent/MeetingComponent.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+  },
+  profile_name: {
+    lineHeight: "2px",
+    verticalAlign: "bottom",
+    paddingLeft: "5px",
+  },
+}));
+
+const MeetingComponent = () => {
+  let meeting = [
```
Comment (ThinkTink) [inline client/src/Component/MeetingComponent/MeetingComponent.js]: const here :smile: I'm a stickler on const

File: client/src/Component/EventBodyComponent/EventBodyComponent.js (hunk 1 above)
Comment (jassalss) [reply client/src/Component/EventBodyComponent/EventBodyComponent.js:11]: Thanks

File: client/src/Component/EventBodyComponent/EventBodyComponent.js (hunk 2 above)
Comment (jassalss) [reply client/src/Component/EventBodyComponent/EventBodyComponent.js]: fixed

File: client/src/Component/MeetingComponent/MeetingComponent.js (hunk 4 above)
Comment (jassalss) [reply client/src/Component/MeetingComponent/MeetingComponent.js]: haha fixed

File: client/src/Component/EventBodyComponent/EventBodyComponent.js (hunk 3 above)
Comment (jassalss) [reply client/src/Component/EventBodyComponent/EventBodyComponent.js]: fixed

**Comment by jassalss** [comment]
Hi, I fixed all other very very small issue on behalf of my team.

**Pull request merged**